src/nidcpower/system_tests/test_system_nidcpower.py

```diff
@@ -724,3 +724,15 @@ def test_wait_for_event_repeated_capabilities(session, channels):
     channels_session = session.channels[channels]
     with channels_session.initiate():
         channels_session.wait_for_event(nidcpower.Event.SOURCE_COMPLETE)
+
+
+@pytest.mark.resource_name("4190/0")
+@pytest.mark.options("Simulate=1, DriverSetup=Model:4190; BoardType:PXIe")
+def test_lcr_attributes(session):
```

Comment:
We typically haven't added unit tests for properties, only for functions.
@sbethur do you think we should start covering properties? Should we go back and ensure full coverage?

Reply:
Good point, although you mean system tests and not unit tests.

I don't think we need simple get/set tests for most of the properties. We need tests for any fancy implementations or to cover any usage of converters.

My preference is to not have these tests. The same applies to tests added in:
 - #1739
 - #1741

Reply:
Having said that, one of the newly added system tests [failed](https://dev.azure.com/ni/DevCentral/_build/results?buildId=2592661&view=logs&j=91f04837-8856-54e7-f3a9-eb021b6dcebe&t=17c95cc2-4630-526b-6703-808c33fdddc8&l=291) 😮

Reply:
The system test failure seems to be due to [the installed NI-DCPower driver version being 21.3](https://dev.azure.com/ni/DevCentral/_build/results?buildId=2592661&view=logs&j=0c6a2735-74d5-5d8e-93ae-f099b0d537de&t=be9a6953-0f60-538e-4637-f21eacf7ef74&l=27) instead of 21.8, which these new LCR-related PRs assume. Is there a way to configure these system tests to be run on NI-DCPower 21.8?

Reply:
I've removed the unneeded attribute system tests in my latest commit.

Reply:
> Is there a way to configure these system tests to be run on NI-DCPower 21.8?

We have an internal work item to update the driver runtimes on system test agents.